components/common-data-items.js

```diff
@@ -319,10 +336,10 @@ export function BiosampleDataItems({
           </DataItemValue>
         </>
       )}
-      {classification && (
+      {classifications.length > 0 && (
```

Comment:
If `classifications` is null, it’ll crash when you do `classifications.length`. You can fix that with `classifications?.length > 0`.

Reply:
even if it's a required property?

Reply:
Is it required? It should say that in the propTypes, and it shouldn’t have a default value on line 311 above.

Reply:
ahh i see it now